rustls/tests/api.rs

```diff
@@ -17,6 +17,10 @@ use rustls::internal::msgs::base::Payload;
 use rustls::internal::msgs::codec::Codec;
 use rustls::internal::msgs::enums::AlertLevel;
 use rustls::internal::msgs::message::PlainMessage;
+use rustls::internal::msgs::{
+    handshake::ClientExtension, handshake::HandshakePayload, message::Message,
+    message::MessagePayload,
+};
```

Comment:
nit: I think the two `rustls::internal::msgs::message::*` imports in this block, and the one on L19 should be combined into their own line, leaving the two `rustls::internal::msgs::handshake::*` imports on their own alongside.

Reply:
good spot - done